ops/README.md

```diff
@@ -0,0 +1,72 @@
+# docker-compose
+
+The docker-compose project runs a local optimism stack.
+
+## prerequisites
+
+- docker
+- docker-compose
+- make
+
+## Starting and stopping the project
+
+The base `docker-compose.yml` file will start the required components for a full stack.
+
+Supplementing the base configuration is an additional metric enabling file, `docker-compose-metrics.yml`. Adding this configuration to the stack will enable metric emission for l2geth and start grafana (for metrics visualisation) and influxdb (for metric collection) instances.
+
+The base stack can be started and stopped with a command like this (there is no need to specify the default docker-compose.yml)
+```
+docker-compose \
+    up --build --detach
+```
+
+To start the stack with monitoring enabled, just add the metric composition file.
+```
+docker-compose \
+    -f docker-compose.yml \
+    -f docker-compose-metrics.yml \
+    up --build --detach
+```
+
+A Makefile has been provided for convience. The following targets are available.
+- make up
+- make down
+- make up-metrics
+- make down-metrics
+
+## Authentication
+
+Influxdb has authentication disabled.
+
+Grafana requires a login. The defaults are:
+```
+user: admin
+password: optimism
+```
+
+## Data persistance
+
+Grafana data is not currently saved. Any modifications or additions will be lost on container restart.
```

Comment:
Should it be?

Reply:
I think it's fine for local development to not be saved; you can be mindful as a developer about keeping the container around if you need it.

Reply:
Easy enough to add and it should prevent some unexpected behavior (who reads docs?).

I'll add this volume.

Reply:
Grafana volume added.